src/main/java/com/conveyal/r5/analyst/scenario/Scenario.java

```diff
@@ -93,11 +95,13 @@ public TransportNetwork applyToTransportNetwork (TransportNetwork originalNetwor
         // Check all the parameters before applying any modifications.
         // Might some parameters may become valid/invalid because of previous modifications in the list?
         List<Modification> badModifications = new ArrayList<>();
```

Comment:
Parallelism: if the next field is called modificationsWithWarnings maybe this one should be renamed modificationsWithErrors.